src/DatePicker/index.js

```diff
@@ -68,10 +72,11 @@ export default class DatePicker {
         <button type="button" class="date-picker__toggle" aria-label="Toggle Date Picker">Toggle Date Picker</button>
 
         <div class="date-picker__calendar" style="display: none">
-          <p class="calendar__header" role="heading"></p>
           <button id="calendar__prev" type="button">${this.props.previousLabel}</button>
           <button id="calendar__next" type="button">${this.props.nextLabel}</button>
+          <p class="calendar__header" role="heading" aria-live="assertive">${this.renderMonthHeader()}</p>
```

Comment:
I'm curious what your reasoning is for putting the heading after the buttons, especially since the heading is `aria-live="assertive"` and will be read on-change.